src/utils/validation.js

```diff
@@ -0,0 +1,26 @@
+import { ERROR_MESSAGES } from "../constants/message.js";
+
+/**
+ * 자동차 이름을 입력했을 때 유효한지 검사합니다.
+ * @param {string} input
+ * @returns
+ */
```

Comment:
각각 함수에 대해 param 설명과 함수 설명이 들어간 점 너무 좋은 것 같아요 !👍
```suggestion
/**
 * 자동차 이름을 입력했을 때 유효한지 검사합니다.
 * @param {string} input
 * @returns
 */
```